app/src/main/java/com/islaharper/dawnofandroid/ui/theme/Theme.kt

```diff
@@ -83,7 +83,7 @@ private val DarkColors = darkColorScheme(
 fun DawnOfAndroidTheme(
     darkTheme: Boolean = isSystemInDarkTheme(),
     // Dynamic color is available on Android 12+
-    dynamicColor: Boolean = true,
+    dynamicColor: Boolean = false,
```

Comment:
Why are we turning off dynamic color?

Reply:
I'm adding option to toggle light/dark mode and another for dynamic on/off. Was setting it to off as the default